core/modules/node/node.module

```diff
@@ -1625,7 +1625,7 @@ function node_menu() {
     'file' => 'node.types.inc',
   );
   $items['admin/structure/types/manage/%node_type/edit'] = array(
-    'title' => 'Edit',
+    'title' => 'Content Type Settings',
```

Comment:
In Backdrop/Drupal, "Sentence casing" is the standard for titles (as you can see in the other menu entries here). So this needs to be `Content type settings`. I've always thought this was a bit silly, but it's the standard and we should stick to it until it is changed everywhere.